<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JSJ_ddmldlg</class>
 <widget class="QMainWindow" name="JSJ_ddmldlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
	
	color: rgb(255, 255, 255);

}
QTableWidget{
	border-radius: 2px; /* 边框圆角 */
border: 1px solid #A0A0A0; /* 边框宽度为1px，颜色为#A0A0A0 */
font: 15pt &quot;黑体&quot;;
background-color: rgb(18,76,106,);
	color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>30</y>
      <width>321</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/jsj/rc/限速命令/缓存命令.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="HC_tableWidget">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>60</y>
      <width>320</width>
      <height>271</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>320</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>280</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QTableWidget" name="Send_tableWidget">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>360</y>
      <width>320</width>
      <height>260</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>320</width>
      <height>260</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>320</width>
      <height>260</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1582</x>
      <y>330</y>
      <width>321</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/jsj/rc/限速命令/发送命令.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>620</y>
      <width>321</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/jsj/rc/限速命令/完成命令.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="Finish_tableWidget">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>650</y>
      <width>320</width>
      <height>261</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>320</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>320</width>
      <height>300</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="cancle_Btn">
    <property name="geometry">
     <rect>
      <x>762</x>
      <y>920</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>160</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>30</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(179, 179, 179);
color: rgb(255, 255, 255);

	border-radius: 6px; /* 边框圆角 */
font: 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>取消命令</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ml_CloseBtn_2">
    <property name="geometry">
     <rect>
      <x>928</x>
      <y>920</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>160</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>30</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 96, 142);
color: rgb(255, 255, 255);

	border-radius: 6px; /* 边框圆角 */
font: 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ml_SendBtn_2">
    <property name="geometry">
     <rect>
      <x>596</x>
      <y>920</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>160</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>30</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 96, 142);
color: rgb(255, 255, 255);

	border-radius: 6px; /* 边框圆角 */
font: 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ml_SaveBtn_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>920</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>160</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>30</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 96, 142);
color: rgb(255, 255, 255);

	border-radius: 6px; /* 边框圆角 */
font: 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>1585</width>
      <height>630</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
QTabBar::tab {

	

 }
QTabWidget QTabBar
{
	font-size: 20px;
}
 QTabBar::tab:selected{
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.278409 rgba(118, 159, 193, 255), stop:1 rgba(255, 255, 255, 255));
}

</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="styleSheet">
      <string notr="true">QWidget#tab{
		background-color: rgba(18, 75, 105, 80);
	font: 14pt &quot;黑体&quot;;
}</string>
     </property>
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="styleSheet">
      <string notr="true">QWidget#tab_2{
	border-image: url(:/new/jsj/rc/限速命令/背景二.png);
	font: 14pt &quot;黑体&quot;;
}</string>
     </property>
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QTableWidget" name="ml_showTable">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>640</y>
      <width>1575</width>
      <height>270</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1575</width>
      <height>270</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1575</width>
      <height>270</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
